tests/test_users.py

```diff
@@ -0,0 +1,208 @@
+import json
+import unittest
+import os, sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from fusillade.clouddirectory import User, Group, Role, ad, cleanup_directory, cleanup_schema
+from tests.common import new_test_directory
+directory = None
+schema_arn = None
+
+
+def setUpModule():
+    global directory, schema_arn
+    directory, schema_arn = new_test_directory()
+
+
+def tearDownModule():
+    global directory, schema_arn
+    cleanup_directory(directory._dir_arn)
+    cleanup_schema(schema_arn)
+
+
+class TestUser(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        with open(f"{pkg_root}/policies/default_user_policy.json", 'r') as fp:
```

Comment:
Again, I think these policy file paths should be centralized and called from somewhere.